### PR TITLE
Feature/hw 64281 add venmo account

### PR DESCRIPTION
**Pull request by shanavas123**
HW 64281 Add Venmo account in core sdk.

### CODE REVIEW

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+ */
+
+public class VenmoAccount extends TransferMethod {
+
+    /**
+     * Construct a {@code VenmoAccount} object from {@link JSONObject} representation
+     *
+     * @param jsonObject raw data information
+     */
+    public VenmoAccount(@NonNull JSONObject jsonObject) throws JSONException {
```
Comment (peter-joseph) [inline core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java:46]: make `jsonObject` `final`

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+        super(jsonObject);
+    }
+
+    /**
+     * Construct a {@code VenmoAccount} object from Map of key-value pair representation
+     *
+     * @param fields map of key-value raw data information
+     */
+    private VenmoAccount(@NonNull Map<String, Object> fields) {
```
Comment (peter-joseph) [inline core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java:55]: make `fields` `final`

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        /**
+         * Constructs a {@code Builder} with predefined attributes
+         *
+         * @param transferMethodCountry  Transfer method country of account
+         * @param transferMethodCurrency Transfer method currency of account
+         * @param accountId              accountId to bind to {@link VenmoAccount}
+         */
+        public Builder(@NonNull String transferMethodCountry,
+                @NonNull String transferMethodCurrency,
+                @NonNull String accountId) {
```
Comment (peter-joseph) [inline core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java:98]: make all params final please

**Review by peter-joseph — COMMENTED** [review]
LGTM just few comments about making parameters `final` so that it it will be easier for the Reflection unit, during JSON to Object conversion

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/TransferMethod.java [hunk 4]
```diff
@@ -245,6 +246,7 @@ protected void setFields(@NonNull final Map<String, Object> fields) {
         public static final String PREPAID_CARD = "PREPAID_CARD";
         public static final String WIRE_ACCOUNT = "WIRE_ACCOUNT";
         public static final String PAYPAL_ACCOUNT = "PAYPAL_ACCOUNT";
+        public static final String VENMO_ACCOUNT="VENMO_ACCOUNT";
```
Comment (vwagh-hw) [inline core/src/main/java/com/hyperwallet/android/model/transfermethod/TransferMethod.java:249]: minor, keep space around `=`

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/TransferMethod.java [hunk 5]
```diff
@@ -321,5 +323,7 @@ protected void setFields(@NonNull final Map<String, Object> fields) {
         public static final String INTERMEDIARY_BANK_POSTAL_CODE = "intermediaryBankPostalCode";
         public static final String INTERMEDIARY_BANK_STATE_PROVINCE = "intermediaryBankStateProvince";
         public static final String WIRE_INSTRUCTIONS = "wireInstructions";
+        //venmo account fields
+        public static final String VENMO_ACCOUNT_ID ="accountId";
```
Comment (vwagh-hw) [inline core/src/main/java/com/hyperwallet/android/model/transfermethod/TransferMethod.java:327]: minor, keep space around `=`

File: core/src/test/java/com/hyperwallet/android/transfermethod/DeactivateVenmoAccountTest.java [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+    public MockitoRule mMockitoRule = MockitoJUnit.rule();
+    @Mock
+    private HyperwalletListener<StatusTransition> mListener;
+    @Captor
+    private ArgumentCaptor<StatusTransition> mStatusTransitionCaptor;
+    @Captor
+    private ArgumentCaptor<HyperwalletException> mExceptionArgumentCaptor;
+
+    @Test
+    public void testDeactivateVenmoAccount_successfulStatusTransition() throws Exception {
```
Comment (vwagh-hw) [inline core/src/test/java/com/hyperwallet/android/transfermethod/DeactivateVenmoAccountTest.java:60]: in my opinion, prepending test methods with `test` seems redundant as this is a test class and method is already annotated as `@Test`

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java (hunk 1 above)
Comment (shanavas123) [reply core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java:46]: fixed.
Screenshot 2020-09-22 at 1 19 49 PM

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java (hunk 2 above)
Comment (shanavas123) [reply core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java:55]: fixed.
Screenshot 2020-09-22 at 1 19 49 PM

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java (hunk 3 above)
Comment (shanavas123) [reply core/src/main/java/com/hyperwallet/android/model/transfermethod/VenmoAccount.java:98]: fixed.
Screenshot 2020-09-22 at 1 23 55 PM

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/TransferMethod.java (hunk 5 above)
Comment (shanavas123) [reply core/src/main/java/com/hyperwallet/android/model/transfermethod/TransferMethod.java:327]: fixed

File: core/src/main/java/com/hyperwallet/android/model/transfermethod/TransferMethod.java (hunk 4 above)
Comment (shanavas123) [reply core/src/main/java/com/hyperwallet/android/model/transfermethod/TransferMethod.java:249]: fixed

File: core/src/test/java/com/hyperwallet/android/transfermethod/DeactivateVenmoAccountTest.java (hunk 6 above)
Comment (shanavas123) [reply core/src/test/java/com/hyperwallet/android/transfermethod/DeactivateVenmoAccountTest.java:60]: All other test method they named prefix as test, for consistency I am follow the same

**Review by peter-joseph — APPROVED** [review]

**Review by vwagh-hw — APPROVED** [review]

**Pull request merged**